db/config.cc

```diff
@@ -686,6 +686,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , replace_address(this, "replace_address", value_status::Used, "", "The listen_address or broadcast_address of the dead node to replace. Same as -Dcassandra.replace_address.")
     , replace_address_first_boot(this, "replace_address_first_boot", value_status::Used, "", "Like replace_address option, but if the node has been bootstrapped successfully it will be ignored. Same as -Dcassandra.replace_address_first_boot.")
     , override_decommission(this, "override_decommission", value_status::Used, false, "Set true to force a decommissioned node to join the cluster")
+    , enable_repair_based_node_ops(this, "enable_repair_based_node_ops", value_status::Used, true, "Set true to use enable repair based node operations instead of streaming based")
```

Comment:
Maybe make this LiveUpdate so we don't have to restart a node to change it.

(on the other hand, I want to remove the configuration item and the old methods)

Reply:
To be able to change the flag on the fly, do you suggest to use the restful api? I think we still need a configuration based way to specify this. For example, when people want to bootstrap a node, with only a restful api, user has to run the api before the bootstrap happens which timing is hard to get. 

If we allow restful api to change the db.config.enable_repair_based_node_ops(), it might surprise the user in a way that the user checks  enable_repair_based_node_ops is true on while someone changes it to false.

Reply:
If you mark the variable as LiveUpdate, then you can change scylla.yaml and SIGHUP the process to re-read it.

Reply:
Ah, I see the new parameter. Thanks!